tests/containers/account/AccountContainer-test.jsx

```diff
@@ -112,15 +112,18 @@ describe('AccountContainer', () => {
         });
     });
 
-    describe('parseBalances', () => {
-        it('should parse the returned balances and add them to the Redux account object', (done) => {
+    describe('parseFYSnapshot', () => {
+        it('should parse the returned fiscal year snapshot and add the data to the Redux account object', (done) => {
             const initialModel = new FederalAccount(mockAccount);
```

Comment:
Should default totals in the [Federal Account model](https://github.com/fedspendingtransparency/usaspending-website/blob/fix-federal-account-v2-endpoint/src/js/models/account/FederalAccount.js#L26) be updated as well?

Reply:
Yep, good catch